src/config/kubeconfig.js

```diff
@@ -10,15 +10,28 @@ function path() {
   if (process.env.KUBECONFIG && process.env.KUBECONFIG.length > 0) {
     const files = process.env.KUBECONFIG.split(pth.delimiter);
     // Only returning the path to the first file if there are multiple.
-    return files[0];
+    if (files.length) {
+      return files[0];
+    }
   }
 
   const home = k8s.findHomeDir();
   if (home) {
-    const cfg = pth.join(home, '.kube', 'config');
-    if (hasAccess(cfg)) {
-      return cfg;
+    const kube = pth.join(home, '.kube');
+    const cfg = pth.join(kube, 'config');
+    if (!hasAccess(cfg)) {
+      if (!hasAccess(kube)) {
+        console.log(`creating dir ${kube}`);
+        fs.mkdirSync(kube);
+      }
+      console.log(`creating file ${cfg}`);
+      fs.writeFileSync(cfg, `\
```

Comment:
Probably better if we did `fs.writeFileSync(cfg, JSON.stringify({apiVersion: 'v1', clusters: []}));` instead; among other things, we can format it better.

Reply:
Good point, I'll make a separate issue for each of the subsequent pristine-machine problems.

Reply:
done